public/js/index.js

```diff
@@ -0,0 +1,24 @@
+//Youtube slideshow for index page
+function onGoogleLoad() {
+  gapi.client.setApiKey('AIzaSyCzI5Gvpq1hPlfjvB81EVc0vLv_11eW7NI');
```

Comment:
You can write some JS to directly extract the key from the file and then save it in a constant e.g.
`var API_KEY = readFile(config.toml)
function onGoogleLoad() {
  gapi.client.setApiKey(API_KEY)
}`

The server shouldn't be serving config.toml so I don't think there is an issue with respect to exposing its location.